trySwift/SpeakerDetailViewController.swift

```diff
@@ -11,7 +11,7 @@ import TrySwiftData
 
 class SpeakerDetailViewController: UITableViewController {
 
-    var speaker: Speaker!
```

Comment:
I suppose we can show an empty `UIViewController` instead of a `SpeakerDetailViewController` when both master and detail are shown. Then, the master can take care of showing the `SpeakerDetailViewController`, passing in the required `Speaker`.